src/app/settings/settings.component.ts

```diff
@@ -61,10 +63,22 @@ export class SettingsComponent extends RouterEnter {
     this.acceptFileTypes = this.filestackService.getFileTypes('image');
     // also get program name
     this.currentProgramName = this.storage.getUser().programName;
-    this.currentProgramImage = this.storage.getUser().programImage;
+    this._getCurrentProgramImage();
```

Comment:
It's better to let `_getCurrentProgramImage()` return the image url and assign it to `this.currentProgramImage` inside this function.

Reply:
ok. I have pushed an update.